flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/observer/SnapshotObserver.java

```diff
@@ -124,14 +159,78 @@ private void observeTriggeredSavepoint(FlinkResourceContext<CR> ctx, String jobI
                         savepointFetchResult.getLocation(),
                         savepointInfo.getTriggerType(),
                         savepointInfo.getFormatType(),
-                        SavepointTriggerType.MANUAL == savepointInfo.getTriggerType()
+                        SnapshotTriggerType.MANUAL == savepointInfo.getTriggerType()
                                 ? resource.getSpec().getJob().getSavepointTriggerNonce()
                                 : null);
 
-        ReconciliationUtils.updateLastReconciledSavepointTriggerNonce(savepointInfo, resource);
+        ReconciliationUtils.updateLastReconciledSnapshotTriggerNonce(
+                savepointInfo, resource, SAVEPOINT);
         savepointInfo.updateLastSavepoint(savepoint);
     }
 
+    /**
+     * Observe the status of triggered checkpoints.
+     *
+     * @param ctx Resource context.
+     * @param jobID the jobID of the observed job.
+     */
+    private void observeTriggeredCheckpoint(FlinkResourceContext<CR> ctx, String jobID) {
```

Comment:
There is nothing in this JavaDoc which would not be obvious by looking at the method signature. I would remove the entire JavaDoc. It's verbose and will get stale during refactors.

Reply:
Introduced for consistency with observeTriggeredSavepoint. Drop it there too?

Reply:
Feel free to remove it there as well.

Reply:
Done.